src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -150,13 +203,16 @@ Void deleteStudyAndAllNeighbors(
     /**
      * Returns a file download containing user app usage data.
      *
+     * @param organizationId         - Id of the organization to which study belongs
      * @param studyId                - the study id
      * @param participantEntityKeyId - the participant entity key id
      * @param fileType               - the type of file (csv, json) to return as the download
      * @return All participant data
      */
-    @GET( BASE + AUTHENTICATED + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + USAGE_PATH )
+    @GET( BASE + AUTHENTICATED + PARTICIPANT_PATH + DATA_PATH + ORGANIZATION_ID_PATH + STUDY_ID_PATH
```

Comment:
This looks like a breaking change.

Reply:
https://github.com/openlattice/chronicle-api/pull/35#discussion_r485306813

Reply:
The only difference is ORGANIZATION_ID_PATH in the new path, and there is no need for backward compatibility